corehq/apps/app_manager/models.py

```diff
@@ -4078,8 +4094,15 @@ class ApplicationBase(VersionedDoc, SnapshotMixin,
     # always false for RemoteApp
     case_sharing = BooleanProperty(default=False)
 
+    build_profiles = SchemaDictProperty(BuildProfile)
+
+    # each language is a key and the value is a list of multimedia referenced in that language
+    media_language_map = SchemaDictProperty(MediaList)
```

Comment:
@calellowitz how come this only contains form media and not media from modules / form menus?